server/src/lib/quotes.ts

```diff
@@ -41,6 +54,32 @@ class Quotes {
 
     return this.quotes[randomIndex];
   }
+
+  /**
+   * Returns groupings of quotes by author
+   *
+   * @returns {Array[Array<Quote>]}
+   */
+  public static getGroup(): Quote[][] {
+    let quoteIndex = 0;
+    const quoteGroupList: Quote[][] = [[], [], []];
+    console.log("quote length =" + this.quotes.length);
+    for (let i = 0; i < 3; i++) {
```

Comment:
This isn't bad to use `for` but it is more common to use `.map` (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map)

https://stackoverflow.com/a/49732119/11505534

You could first define the size of the array and then run it through map to generate the proper value.
```
let size = 3
[...Array(size)].map((_)=> { return ... calculate quotes here and then return } );
```

It is common to use map because people have found it cleaner and more semantically easy to understand. Give it a shot to reimplment this using `.map`